community-membership.md

```diff
@@ -160,73 +159,67 @@ The following apply to the part of codebase for which one would be an approver i
+  - Authoring and reviewing proposals
+  - Initiating, contributing and resolving discussions (emails, GitHub issues, meetings)
+  - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the area through either implementation or review
 
 ### Responsibilities and privileges
 
 The following apply to the area / component for which one would be an owner.
 
-- Owner status may be a precondition to accepting a new component or piece of major functionality
-- Design/proposal approval authority over the area / component, though escalation to [`kubernetes-maintainers`](https://groups.google.com/forum/#!forum/kubernetes-maintainers) is possible.
-- Mentor and guide approvers, reviewers, and members.
-
-## Maintainer
-
-[**Kubernetes project maintainers**](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers) work holistically
-across the project to maintain its health and success.  They are typically involved in multiple different areas, and
-have made substantial contributions both through code and broader organizational efforts.
-
-**Defined by:** *write* access to the kubernetes GitHub repo
-
-TODO: Determine if this role is outdated and needs to be redefined or merged into owner role.
-
-### Requirements
-
-- Approver for some part of the codebase for at least 3 months
-- Member for at least 1 year
-- Primary reviewer for 20 substantial PRs
-- Reviewed or merged at least 50 PRs
-- Apply to [`kubernetes-maintainers`](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers), with:
-  - A [Champion](/incubator.md#faq) from the existing
-    kubernetes-maintainers members
-  - A Sponsor from Project Approvers
-  - Summary of contributions to the project
-  - Current project responsibilities
-  - Links to merged and assigned PRs
-- At least 3 of the maintainers must approve the application, with no objections
-- Application expires after 2 weeks if not enough approvals are granted
-
-### Responsibilities and privileges
-
-- Write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki, create/delete labels and milestones)
-- Capable of directly applying lgtm + approve labels for any PR, causing it to be merged by the submit queue - but permitted only in rare instances
-   - Expected to respect OWNERS files approvals and use standard procedure for merging code
-- Expected to work to holistically maintain the health of the project through:
-  - Reviewing PRs
-  - Fixing bugs
-  - Providing user support
-  - Mentoring and guiding approvers, reviewers, and other contributors
+- Make and approve technical design decisions for the area.
+- Set technical direction and priorities for the area.
+- Define milestones and releases.
+- Mentor and guide approvers, reviewers, and contributors to the area.
+- Ensure continued health of area
+  - Adequate test coverage to confidently release
+  - Tests are passing or are fixed when they fail
```

Comment:
passing reliably (i.e., not flaky)

Reply:
We need a document that enumerates expected development practices, including testing, documentation, release notes, release deadlines (e.g., feature complete), PR reviews, issue triage, etc. For now, I'd just convey that the owners are ultimately responsible for the subproject.

Reply:
+1.  My hope is that this will be covered in the charter for the subproject and the charter templates will provide a set of accepted practices.